app/views/layouts/application.html.erb

```diff
@@ -9,7 +9,30 @@
     <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
   </head>
 
-  <body>
+<body>
+  <div id="flex-content-container">
```

Comment:
what is flex-content-container used for?

Reply:
Flex is used in bootstrap to allow the nav to adjust to different screen sizes.

Reply:
> what is flex-content-container used for?

Flex is used in bootstrap to allow the nav to adjust to different screen sizes. @liangma87